src/state.py

```diff
@@ -0,0 +1,60 @@
+from abc import ABC, abstractmethod
+
+
+"""
+Game state could be changed from the <Context> object as well as from any <BaseState> descendant
+"""
+
+
+class Context(ABC):
```

Comment:
1. please move state.py to src/core/states
2. move base state class and others to separate files to make main.py much more clear

Reply:
OK